### PR TITLE
Add st_things build configuration

### PR DESCRIPTION
**Pull request by heejin-kim**
It is used to build st_things frameworks and samples.

### CODE REVIEW

**Comment by heejin-kim** [comment]
It should be changed because this repository folders is different with my repo. I'm checking that.

**Comment by heejin-kim** [comment]
I revised the Kconfig file

File: build/configs/artik053/st_things/Make.defs [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+############################################################################
+# configs/artik053/st_things/Make.defs
```
Comment (davidfather) [inline build/configs/artik053/st_things/Make.defs]: Why do we need nuttx license for this file?

Comment (davidfather) [inline build/configs/artik053/st_things/Make.defs]: Why do we need nuttx license for this file?

Comment (heejin-kim) [reply build/configs/artik053/st_things/Make.defs]: modified

**Pull request merged**